packages/async-flow/src/equate.js

```diff
@@ -69,7 +69,7 @@ export const makeEquate = bijection => {
         return;
       }
       case 'tagged': {
-        equate(g.tag, h.tag, 'tag');
+        equate(getTag(g), getTag(h), 'tag');
```

Comment:
Genuine bug I was trying to get the tag with specimen.tag, rather than the getTag function. Found similar bug in convert by testing, and then this by looking for other occurrences. Not caught by testing because `undefined === undefined`. Fixed.